dbms/src/test/java/org/polypheny/db/jdbc/JdbcAutoCommitTest.java

```diff
@@ -82,7 +82,7 @@ public void testDDl() throws SQLException {
                                 statement2.executeQuery( "SELECT * FROM autoCommit" ), ImmutableList.of() );
                     }
                 } finally {
-                    statement.executeUpdate( "DROP TABLE IF EXISTS autocomit" );
+                    statement.executeUpdate( "DROP TABLE IF EXISTS autoCommit" );
```

Comment:
There is, in fact, a typo (it misses an "m" in commit). However, the casing is valid since relational namespaces are case-insensitive per default. Is there a particular reason for adjusting the casing in the Drop DDL?

Reply:
Yes, it is spelled with a capital C everywhere else in the file, just making it consistent.